### PR TITLE
Do PoC on docker doc build with intersphinx using mounted index build

### PR DESCRIPTION
**Pull request by Almad**
Pull request for [Do PoC on docker doc build with intersphinx using mounted index build](https://trello.com/c/N3wnPCwH/272-do-poc-on-docker-doc-build-with-intersphinx-using-mounted-index-build).

### CODE REVIEW

**Comment by Almad** [comment]
This is mainly for @nadade for demonstration, but also //cc @JackuB, @smizell, @honzajavorek (and maybe @kylef) as I feel they may be interested ;)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
-# example-intersphinx-repo
-This repository demonstrates using Intersphinx with indexes being exported in Docker volume
+# Example Intersphinx Repository
+
+This repository demonstrates a possible approach for maintaining a collection internal, cross-referenced documentations.
```
Comment (honzajavorek) [inline README.md]: collection _of_ internal

File: README.md [hunk 2]
```diff
@@ -1,2 +1,45 @@
...
+Idea revolves around having a shared repository with all inventories (indexes) for all projects. Those can then be reused and referenced by other documentations. Index is updated after build (every project must update its inventory file) and stored in a docker repository to reuse build and authentication structure that's already in place for it.
+
+## Try it out
+
+* `./build.sh`
+* `open ./example-intersphinx-project/build/html/index.html`
+
+## Repository Structure
+
+* `example-standalone-docs` contains minimal spinx documentation with glossary that defines a term
```
Comment (honzajavorek) [inline README.md]: spinx -> Sphinx

**Comment by nadade** [comment]
@Almad I've successfully ran the script and produced output, I think it's good to go :+1: 

sidenote: I'd vote for more emphasis on using namespaces with the common terms to avoid duplicity and confusion

File: example-intersphinx-project/source/index.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Welcome to Example Intersphinx Project's documentation!
+=======================================================
+
+In this project, we rely in definition of :term:`Example Term`, as provided by Example Standalone Docs.
+
+
+If we want to future proof, we can reference :term:`intertest:Example Term` as a namespaced term (see source code).
```
Comment (honzajavorek) [inline example-intersphinx-project/source/index.rst]: want to -> want to be

**Comment by honzajavorek** [comment]
Finished my typo hunt. Minor stuff.

Good we have intersphinx implementation! :+1: Didn't try, I trust @nadade. I'm happy, this will be useful. On the other hand - will it be? Do we plan to have more doc projects than internal docs as they are now? Security Whitepaper? Core App docs?

**Comment by nadade** [comment]
@honzajavorek I think every library should have its docs eventually plus even now there are various terms used for the same thing throughout the documentation, this will definitely help keep things tidy and coherent..

**Comment by Almad** [comment]
@honzajavorek yes. every service should have its own docs. `apiary` has it even now, but it'll be more widespread, hopefully :)

File: README.md (hunk 1 above)
Comment (Almad) [inline README.md]: fixd

File: README.md (hunk 2 above)
Comment (Almad) [inline README.md]: fixd

**Pull request merged**